lib/vsc/mympirun/mpi/intelmpi.py

```diff
@@ -173,7 +176,7 @@ def pinning_override(self):
         if self.pinning_override_type in ('packed', 'compact', 'bunch'):
             cmd.add(['-env', 'I_MPI_PIN_PROCESSOR_LIST=allcores:map=bunch'])
         elif self.pinning_override_type in ('spread', 'scatter'):
-            cmd.add(['-env', 'I_MPI_PIN_PROCESSOR_LIST=allcores'])
+            cmd.add(['-env', 'I_MPI_PIN_PROCESSOR_LIST=allcores:map=%s' % self.pinning_override_type])
```

Comment:
what are the possible values?

Reply:
https://software.intel.com/en-us/mpi-developer-reference-windows-environment-variables-for-process-pinning

bunch, spread and scatter

Reply:
tests?